apps/mobile/src/navigation/PostStackNavigator.tsx

```diff
@@ -0,0 +1,26 @@
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
+
+import { PostStackNavigatorParamList } from '~/navigation/types';
+import { PostScreen } from '~/screens/PostScreen/PostScreen';
+import { ProfilePicturePickerScreen } from '~/screens/ProfilePicturePickerScreen/ProfilePicturePickerScreen';
+
+const Stack = createNativeStackNavigator<PostStackNavigatorParamList>();
+
+function Empty() {
+  return null;
+}
+
+export function PostStackNavigator() {
+  return (
+    <Stack.Navigator screenOptions={{ header: Empty }} initialRouteName="NftSelector">
+      <Stack.Screen
+        name="NftSelector"
+        component={ProfilePicturePickerScreen}
```

Comment:
let's rename `ProfilePicturePickerScreen` to be more general since we're using it for the Post flow now as well

Reply:
maybe `NftSelectorScreen`

Reply:
same for some components inside `ProfilePicturePickerScreen` like `ProfilePicturePickerFilterBottomSheet` and `ProfilePicturePickerGrid`